spec/CloudCode.Validator.spec.js

```diff
@@ -860,6 +860,60 @@ describe('cloud validator', () => {
       });
   });
 
+  it('basic validator requireUserRole', async function (done) {
+    Parse.Cloud.define(
+      'cloudFunction',
+      () => {
+        return true;
+      },
+      {
+        requireUser: true,
+        requireUserRole: ['Admin'],
+      }
+    );
+    const user = await Parse.User.signUp('testuser', 'p@ssword');
+    try {
+      await Parse.Cloud.run('cloudFunction');
+      fail('cloud validator should have failed.');
+    } catch (e) {
+      expect(e.message).toBe('Validation failed. User does not match the required roles.');
+    }
+    const roleACL = new Parse.ACL();
+    roleACL.setPublicReadAccess(true);
+    const role = new Parse.Role('Admin', roleACL);
+    role.getUsers().add(user);
+    await role.save({ useMasterKey: true });
+    await Parse.Cloud.run('cloudFunction');
+    done();
+  });
+
+  it('basic string requireUserRole', async function (done) {
+    Parse.Cloud.define(
+      'cloudFunction',
+      () => {
+        return true;
+      },
+      {
+        requireUser: true,
+        requireUserRole: 'Admin',
+      }
+    );
```

Comment:
Personally i'm not a huge fan of supporting dual data type  (string/array). Some developers might think that it can only require one role. Renaming the key to `requireUserRoles` and only supporting the Array of string will be more clear on the usage and into the docs :)